### PR TITLE
changed html so it displays a message when you dont have any tickets yet

### PR DESCRIPTION
**Pull request by juliaf1**
Hey, guys! I added elses statements on our tickets index so it displays a message with a link when a person doesn't have tickets booked yet, instead of empty containers.

Captura de Tela 2020-11-27 às 11 29 28

### CODE REVIEW

**Comment by rowan2992** [comment]
Look GREAT
Typo: change Ops to Oops

File: app/views/tickets/index.html.erb [hunk 1]
```diff
@@ -77,6 +84,13 @@
           </div>
         <% end %>
       </div>
+
+    <% else %>
+      <div class="card-title mt-5 px-2">
+        <h2 class="name-bold">Tickets as a Teacher</h2>
+        <p>Looks like you're not teaching anyone yet.</p>
+        <%= link_to 'Add your available timeslots so people can book you', new_timeslot_path %>
```
Comment (tvbgo) [inline app/views/tickets/index.html.erb:92]: Do we need to pass the user here, or is this link working?

Comment (juliaf1) [reply app/views/tickets/index.html.erb:92]: link is working!

**Comment by juliaf1** [comment]
@rowan2992 changing the typo, thankss! 🥇

**Pull request merged**